recipes/pysparse/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pysparse" %}
+{% set version = "1.3-dev" %}
+{% set sha256 = "99066f3790fd38ff9f76ae2b83e36c29f423057ab3747178466511e7d9e0adb4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  # PyPI doesn't build
+  patches:
+    - brokenpip.patch # the patch file is expected to be found in the recipe
```

Comment:
Could you please write a brief explanation why this patch is necessary?

Reply:
Explanation added